### ISSUE TITLE
Retrieve number prefix for currently selected country?

### ISSUE THREAD

**Issue by reintroducing** [catamphetamine/react-phone-number-input#284]
Is there a good way (or any way?) to retrieve the number prefix of the currently selected country? For instance, if I have the US flag selected, I'd like to show "+1" somewhere outside of the input. Or the UK with "+44".

Is this currently supported?

**Comment by catamphetamine** [comment]
See `getCountryCallingCode()` in `libphonenumber-js`.

**Comment by reintroducing** [comment]
@catamphetamine fantastic! thanks for the quick response. Do you expose this through this component in an export or do i have to install and manage that as a dependency separately?

**Comment by catamphetamine** [comment]
Hmm.
Perhaps I should: people keep asking for it.

**Comment by reintroducing** [comment]
i think that'd be great. it would lessen one more dep that you've gotta update in package.json of your own project and since its a dependency of this project anyway (and thus already installed), just passing it through as an export in index.js would be fantastic.

**Comment by reintroducing** [comment]
actually, i just looked at all the files, and this works:
`import {getCountryCallingCode} from 'react-phone-number-input/libphonenumber';`

**Comment by catamphetamine** [comment]
```js
// Deprecated
console.warn('Importing from "react-phone-number-input/libphonenumber" is deprecated. Use "libphonenumber-js" directly instead.')
module.exports = require('libphonenumber-js/index.common.js')
```

**Comment by reintroducing** [comment]
lol, touche. can you please expose it? :)

**Comment by catamphetamine** [comment]
Released `react-phone-number-input@2.3.23`
https://github.com/catamphetamine/react-phone-number-input#utility

**Comment by reintroducing** [comment]
Thanks for this one, i tested it and it works as expected, appreciate it.
